### PR TITLE
feat: Add keyGet function and related test.

### PR DESCRIPTION
**Pull request by AsakusaRinne**
Add ```keyGet``` function and related test.

### CODE REVIEW

**Comment by hsluoyz** [comment]
@sagilio

File: Casbin/Util/BuiltInFunctions.cs [hunk 1]
```diff
@@ -15,6 +15,25 @@ public static class BuiltInFunctions
...
+        /// </summary>
+        /// <param name="key1">The first argument.</param>
+        /// <param name="key2">The second argument.</param>
+        /// <returns>Whether key1 matches key2.</returns>
+        public static string KeyGet(string key1, string key2)
+        {
+            int m = key1.Length, n = key2.Length;
+            if(m < n) return "";
+            for (int i = 0; i < n; i++){
+                if(key2[i] == '*') return key1.Substring(i, m - i);
```
[the remark this answers is not in the document]
Comment (AsakusaRinne) [reply Casbin/Util/BuiltInFunctions.cs]: Ok, I have modified it.

File: Casbin/Util/BuiltInFunctions.cs [hunk 2]
```diff
@@ -15,6 +15,28 @@ public static class BuiltInFunctions
...
+        /// KeyGet returns the matched part
+        /// For example, "/foo/bar/foo" matches "/foo/*"
+        /// "bar/foo" will been returned
+        /// </summary>
+        /// <param name="key1">The first argument.</param>
+        /// <param name="key2">The second argument.</param>
+        /// <returns>Whether key1 matches key2.</returns>
+        public static string KeyGet(string key1, string key2)
+        {
+            int m = key1.Length, n = key2.Length;
```
Comment (sagilio) [inline Casbin/Util/BuiltInFunctions.cs:28]: `m` and `n` may not be the most suitable name, It is difficult to see the association with key1 or key2.

**Review by sagilio — APPROVED** [review]
LGTM

**Pull request merged**